### PR TITLE
Model Adapter CICD 

### PR DESCRIPTION
**Pull request by XinyaoWa**

### CODE REVIEW

File: .github/workflows/integration_test_ma_baseline_ddp.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        submodules: recursive
+
+    - name: Build Docker
+      run: |
+        cd Dockerfile-ubuntu18.04 && docker build -t e2eaiok-pytorch112 . -f DockerfilePytorch112 && cd .. && yes | docker container prune && yes | docker image prune
+
+    - name : Test Model Adapter Baseline DDP
+      run: |
+        if [[ $(docker ps -q --filter "name=test-ma-baseline-ddp" | wc -l) -ne 0 ]]; then docker ps -q --filter "name=test-ma-baseline-ddp" | grep -q . && docker stop $(docker ps -a -q) && docker rm $(docker ps -a -q); fi
+        docker run --rm --name test-ma-baseline-ddp --shm-size=10g --privileged --network host --device=/dev/dri -v /mnt/DP_disk1/dataset/MA:/home/vmagent/app/data  -v `pwd`:/home/vmagent/app/e2eaiok -w /home/vmagent/app/ e2eaiok-pytorch112 /bin/bash -c ". /home/vmagent/app/e2eaiok/tests/cicd/jenkins_ma_baseline_ddp_test.sh"
```
Comment (Jian-Zhang) [inline .github/workflows/integration_test_ma_baseline_ddp.yml:33]: Did you hard-coded the # of nodes in ddp_test.sh? will it be a problem if the testing env is smaller?

File: .github/workflows/integration_test_ma_finetuner_autotune.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: Integration Test Model Adapter finetuner autotune
```
Comment (Jian-Zhang) [inline .github/workflows/integration_test_ma_finetuner_autotune.yml:1]: Differences between fintuner & autotune?

File: conf/ma/demo/baseline/cifar100_res18.yaml [hunk 3]
```diff
@@ -7,7 +7,7 @@ train_epochs: 1
 
 data_set: "cifar100"
 data_path:  "/home/vmagent/app/data/dataset/cifar"
-num_workers: 4
+num_workers: 0
```
Comment (Jian-Zhang) [inline conf/ma/demo/baseline/cifar100_res18.yaml:10]: is 0 correct？

File: .github/workflows/integration_test_ma_baseline.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+name: Integration Test Model Adapter baseline
```
[the remark this answers is not in the document]
Comment (XinyaoWa) [reply .github/workflows/integration_test_ma_baseline.yml]: Have removed.

File: .github/workflows/integration_test_ma_baseline_ddp.yml (hunk 1 above)
Comment (XinyaoWa) [reply .github/workflows/integration_test_ma_baseline_ddp.yml:33]: in ddp_test.sh, we start two processes on one node for testing, thus no worry for env change.

File: .github/workflows/integration_test_ma_finetuner_autotune.yml (hunk 2 above)
Comment (XinyaoWa) [reply .github/workflows/integration_test_ma_finetuner_autotune.yml]: Before autotune is for hyperparameter tuning, now has been removed.

File: conf/ma/demo/baseline/cifar100_res18.yaml (hunk 3 above)
Comment (XinyaoWa) [reply conf/ma/demo/baseline/cifar100_res18.yaml:10]: Yes, 0 is torch default value for num_worker.

**Comment by xuechendi** [comment]
@XinyaoWa , pls resolve this conflict

**Pull request closed without merging**